quarkus/extensions/pom.xml

```diff
@@ -34,15 +34,15 @@
         </dependency>
         <dependency>
             <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-server-spi-private</artifactId>
+            <artifactId>keycloak-services</artifactId>
```

Comment:
Looks like you are just rearranging things here, is there a reason for it?

Reply:
`server-spi-private`is duplicated.